vms/platformvm/block/executor/proposal_block_test.go

```diff
@@ -392,57 +392,52 @@ func TestBanffProposalBlockUpdateStakers(t *testing.T) {
 	// Staker5:                                     |--------------------|
 
 	// Staker0 it's here just to allow to issue a proposal block with the chosen endTime.
-	staker0RewardAddress := ids.ShortID{2}
+
+	// In this test multiple stakers may join and leave the staker set at the same time.
+	// The order in which they do it is asserted; the order may depend on the staker.TxID,
+	// which in turns depend on every feature of the transaction creating the staker.
+	// So in this test we avoid ids.GenerateTestNodeID, in favour of ids.BuildTestNodeID
+	// so that TxID does not depend on the order we run tests.
 	staker0 := staker{
-		nodeID:        ids.NodeID(staker0RewardAddress),
-		rewardAddress: staker0RewardAddress,
+		nodeID:        ids.BuildTestNodeID([]byte{0xf0}),
+		rewardAddress: ids.BuildTestShortID([]byte{0xf0}),
```

Comment:
```suggestion
		rewardAddress: ids.ShortID{0xf0},
```
Seems simpler no? (same below)

Reply:
It is. Dropped the helper